app/utils/email_utils.py

```
@@ -1,8 +1,22 @@
-    mail.send(msg)
\ No newline at end of file
+import smtplib
+from email.message import EmailMessage
+from app.config import Config
+
+
+def send_email(subject, receiver, message):
+    if isinstance(receiver, (list, tuple, set)):
+        receiver_email = ", ".join(receiver)
+    else:
+        receiver_email = str(receiver)
```

Ruff (0.8.2)

Local variable `receiver_email` is assigned to but never used

Remove assignment to unused variable `receiver_email`

(F841)